Hcal/testbeam04-2022/pedestals/testbeam22_missing_pedestals.ipynb

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
Thank you for sharing this, I will take a look, but we will probably drop this before merging to keep the git repo clean and focused on just the data tables.